pt-BR/mail/chrome/messenger/credits.dtd

```diff
@@ -1,7 +1,7 @@
 <!ENTITY credit.title     "Créditos do &brandFullName;">
-<!ENTITY credit.leads     "Engineering Leads">
-<!ENTITY credit.core      "Core Development Posse">
-<!ENTITY credit.gecko     "Gecko Layout Engine">
+<!ENTITY credit.leads     "Engenheiros-chefes">
```

Comment:
CORRECTIONS NEEDED it should be !ENTITY credit.leads     "Responsáveis pela engenharia">